pandas/core/indexes/multi.py

```diff
@@ -288,6 +288,26 @@ def _verify_integrity(self, labels=None, levels=None):
     def levels(self):
         return self._levels
 
+    @property
+    def _is_homogeneous(self):
+        """Whether the levels of a MultiIndex all have the same dtype.
```

Comment:
can u share docstrings at all?

Reply:
I think they're all different enough that sharing would be burdensome.